### PR TITLE
Remove uijson action

### PR DESCRIPTION
**Pull request by dannymandel**
@smrgeoinfo @datadavev now that the UIJSON lives with the deployed iSamples in a Box instance, there is no need to include the action that generates it whenever the vocabularies change.

### CODE REVIEW

File: .github/actions/github_action_main.py [hunk 1]
```diff
@@ -19,6 +19,7 @@
 
 
 PATH_PREFIX = "/app/"
+VOCAB_PY = f"{PATH_PREFIX}/tools/vocab.py"
```
Comment (dannymandel) [inline .github/actions/github_action_main.py:22]: This was an unrelated change I had lying around when testing things out.  If you are trying to run the `vocab.py` not inside the docker container, this will just let you adjust the `PATH_PREFIX` in one spot and have everything work.  For instance, if you have the repository checked out you just set the `PATH_PREFIX` to your checkout location.

File: .github/actions/github_action_main.py [hunk 2]
```diff
@@ -70,11 +71,8 @@ def main():
 
 
     if command == "uijson":
-        print("Generating uijson for inclusion in webUI build")
-        index = 0
-        while index < len(inputttl):
-            _run_uijson_in_container(os.path.join(path, inputttl[index]+".json"), inputvocaburi[index])
-            index += 1
+        print("uijson action has been removed.  json is now fetched dynamically at page load.")
+        sys.exit(-1)
```
Comment (dannymandel) [inline .github/actions/github_action_main.py:75]: Leaving this in for failure purposes, in case there is a usage I forgot.  This would let anything still calling this action get a big red failure in the GitHub UI.

**Review by datadavev — APPROVED** [review]
No issues - looks clean
